README.md

```diff
@@ -21,27 +21,43 @@ Atul Deshpande, Li-Fang Chu, Ron Stewart, Anthony Gitter.
 This code requires the glmnet_matlab package (http://web.stanford.edu/~hastie/glmnet_matlab/download.html).
 Unzip `glmnet_matlab.zip` in either the root directory (that contains `SCINGE_Example.m`) or the `code` subdirectory.
 
+
+## Modes of Execution
+SINGE can be executed in the following ways:
+
+**MATLAB environment**
+```
+SINGE(Data,gene_list,outdir,hyperparameter_file)
+```
+**Compiled MATLAB code with R2018a runtime**
+```
+sh standalone_SINGE.sh Data gene_list outdir hyperparameter_file runtime_dir
+```
+
 ## Inputs
 - *Data* - Path to matfile with ordered single-cell expression data (`X`), pseudotime values (`ptime`), and optional indices of regulators (`regix`) (e.g., `data1/X_SCODE_data.mat`)
+- *gene_list* - Path to file containing list of gene names corresponding to the rows in the expression data matrix `X` in Data (e.g., `data1/tf.mat`)
 - *outdir* - Path to folder for storing results from individual GLG Tests
 - *num_replicates* - Number of subsampled replicates obtained for each GLG Test
-- *gene_list* - Path to list of gene names corresponding to the rows in the expression data matrix `X` in Data (e.g., `data1/tf.mat`)
-- *param_list* - A list of GLG hyperparameter combinations for the hyperparameters described below
+- *hyperparameter_file* - Path to file containing a list of GLG hyperparameter combinations for the hyperparameters described below
+
+**Additional input for compiled MATLAB code with R2018a runtime**
+- *runtime_dir* - Path to MATLAB R2018a runtime library
 
 **GLG Hyperparameters:**
-- *param.ID* - Identifier for GLG hyperparameter set
```

Comment:
Let's add a reference to the default or example hyperparameters file so that users see what the expected file format looks like.